src/styles/index.scss

```diff
@@ -1,3 +1,162 @@
+@import './variables';
+
+// Міксин для центрованих елементів
+@mixin flex-center($width, $height) {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: $width;
+  height: $height;
+}
+
+@mixin star-position {
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+// Міксин для стилю тексту
+@mixin text-style($font-size, $font-weight, $color) {
+  font-size: $font-size;
+  font-weight: $font-weight;
+  color: $color;
+}
+
+html {
+  font-family: $font-family;
+}
+
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  width: 198px;
+  height: 406px;
+
+  &__content {
```

Comment:
Avoid setting the same width and height for an element and its BEM element. If `.card` has a set width and height, the `&__content` should not repeat those properties unless it is intentionally different.